addons/website_payment/views/res_config_settings_views.xml

```diff
@@ -15,16 +15,34 @@
                         documentation="/applications/finance/payment_providers.html"
                     >
                         <div class="content-group">
-                            <div class="row" invisible="providers_state == 'other_than_paypal'">
+                            <div class="row" invisible="providers_state">
                                 <field name="providers_state" invisible="1"/>
-                                <field name="is_stripe_supported_country" invisible="1"/>
-                                <div class="oe_inline"
-                                     invisible="not is_stripe_supported_country">
-                                    <button string="Activate Stripe" name="action_activate_stripe" type="object" class="btn-primary"/>
+                                <div
+                                    class="oe_inline"
+                                    invisible="onboarding_payment_provider != 'stripe'"
+                                >
+                                    <button
+                                        string="Activate Stripe"
+                                        name="action_activate_payment_provider"
+                                        type="object"
+                                        class="btn-primary"
+                                    />
+                                </div>
+                                <div
+                                    class="oe_inline"
+                                    invisible="onboarding_payment_provider != 'razorpay'"
+                                >
+                                    <button
+                                        string="Activate Razorpay"
+                                        name="action_activate_payment_provider"
+                                        type="object"
+                                        class="btn-primary"
+                                    />
                                 </div>
                                 <div class="oe_inline"
                                      title="Stripe Connect is not available in your country, please use another payment provider."
-                                     invisible="is_stripe_supported_country">
+                                     invisible="onboarding_payment_provider in ('stripe',
+                                        'razorpay')">
```

Comment:
Nit: adapt as other Not related to your changes but might be good opportunity

Also purpose of 100 char in on line to increase readability and make code look prettier but if we have like only one attribute longer like class attribute in `website_sale` or here tittle and invisible attribute here which will look better in single line even with >100 as dividing it into multiple line will not any better then it's okay to keep it in one line :)